database/migrations/2024_08_08_122117_refactor_created_deleted_updated_by_columns.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\QueryException;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration
+{
+    public function up(): void
+    {
+        $activitiesTableName = config('activitylog.table_name');
+        foreach (Relation::morphMap() as $morphAlias => $class) {
```

Comment:
this is quite dangerous. If this migration is run at different times, this creates different outcomes.